web/src/features/moreCast2/components/TabbedDataGrid.tsx

```diff
@@ -433,8 +431,8 @@ const TabbedDataGrid = ({ morecast2Rows, fromTo, setFromTo, fetchWeatherIndeterm
       // forecast field value with the value of the cell that was double-clicked
       row[forecastField].choice = headerName
       row[forecastField].value = params.value
-      // We've updated local state directly, so now we have to re-render by calling setVisibleRows
-      setVisibleRows([...visibleRows])
+      // We've updated local state directly, so now we need to sync Redux state
+      dispatch(storeUserEditedRows([...visibleRows]))
```

Comment:
I think this actually should be a dispatch to `getSimulatedIndicesAndStoreEditedRows` in order to trigger the simulated indices refresh. Might be a couple other places we need to do this as well.